src/main/java/frc/robot/RobotContainer.java

```diff
@@ -330,6 +332,10 @@ private void configureAutoSelector() {
     SmartDashboard.putData(autoChooser);
   }
 
+  public Command checkForManualZeroing() {
+    return new ManualZeroElevator(subElevator);
+  }
+
```

Comment:
You created the method correctly but it needs to be called

See last year's code
https://github.com/FRCTeam3255/2024_Offbot_Code/blob/dbdd2afe963247d7a9535f7752eb58f633c5b246/src/main/java/frc/robot/Robot.java#L65-L67

https://github.com/FRCTeam3255/2024_Offbot_Code/blob/dbdd2afe963247d7a9535f7752eb58f633c5b246/src/main/java/frc/robot/Robot.java#L113

Also it should be set to ignore disabled
https://github.com/FRCTeam3255/2024_Offbot_Code/blob/dbdd2afe963247d7a9535f7752eb58f633c5b246/src/main/java/frc/robot/RobotContainer.java#L408-L409